tests/network/l2_bridge/libl2bridge.py

```
@@ -216,7 +226,18 @@ def set_secondary_static_ip_address(
     # The function fails on timeout if the interface or its address are not found,
     # so there's no need to check its return code.
-    hot_plugged_interface_ip = lookup_iface_status_ip(vm=vm, iface_name=vmi_interface.name, ip_family=4)
+    try:
+        hot_plugged_interface_ip = lookup_iface_status_ip(vm=vm, iface_name=vmi_interface.name, ip_family=4)
+    except IpNotFound:
+        if is_jira_open(jira_id="CNV-77961"):
+            hot_plugged_interface_ip = read_guest_interface_ipv4(vm=vm, interface_name=vmi_interface.interfaceName).ip
+            expected_ip = ipaddress.IPv4Address(address=ipv4_address)
+            LOGGER.warning(
+                f"CNV-77961: Verified IP {hot_plugged_interface_ip} on {vmi_interface.name} via console "
+                f"(guest-agent not reporting on VM {vm.name})."
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)